include/sd_sector_addresses.h

```diff
@@ -22,122 +22,251 @@ struct SectorAddress
+#define HOME_SSMENU_SELECTED SectorAddress { 0x0000, 0x2098 }
+#define HOME_SSMENU_ACTIVATED SectorAddress { 0x0000, 0x20E9 }
 
-#define SIDE_STEP_RIGHT_NORMAL SectorAddress { 0x0000, 0x2D90 }
-#define SIDE_STEP_RIGHT_NORMAL_SELECTED SectorAddress { 0x0000, 0x2E32 }
-#define SIDE_STEP_RIGHT_NORMAL_ACTIVATED SectorAddress { 0x0000, 0x0561 }
+/**********
+ * Walk menu + home stand + sit + home
+ **********/
+#define WALK_4 SectorAddress { 0x0000, 0x31FF }
+#define WALK_4_SELECTED SectorAddress { 0x0000, 0x171A }
+#define WALK_4_ACTIVATED SectorAddress { 0x0000, 0x176B }
 
-// Single steps
-#define SINGLE_STEP_SMALL SectorAddress { 0x0000, 0x097E }
-#define SINGLE_STEP_SMALL_SELECTED SectorAddress { 0x0000, 0x0A20 }
-#define SINGLE_STEP_SMALL_ACTIVATED SectorAddress { 0x0000, 0x09CF }
+#define HOMESTAND_4_WALKMENU SectorAddress { 0x0000, 0x3250 }
+#define HOMESTAND_4_WALKMENU_SELECTED SectorAddress { 0x0000, 0x180D }
+#define HOMESTAND_4_WALKMENU_ACTIVATED SectorAddress { 0x0000, 0x185E }
 
-#define SINGLE_STEP_NORMAL SectorAddress { 0x0000, 0x0747 }
-#define SINGLE_STEP_NORMAL_SELECTED SectorAddress { 0x0000, 0x07E9 }
-#define SINGLE_STEP_NORMAL_ACTIVATED SectorAddress { 0x0000, 0x0798 }
+#define SIT_4_WALKMENU SectorAddress { 0x0000, 0x315D }
+#define SIT_4_WALKMENU_SELECTED SectorAddress { 0x0000, 0x1900 }
+#define SIT_4_WALKMENU_ACTIVATED SectorAddress { 0x0000, 0x1951 }
 
-#define SINGLE_STEP_LARGE SectorAddress { 0x0000, 0x0654 }
-/***** Walk menu *****/
+#define HOME_WAlKMENU SectorAddress { 0x0000, 0x31AE }
 
 /**********
- * Sit menu
+ * Single step menu + home stand + sit + home
  **********/
-#define SIT SectorAddress { 0x0000, 0x222C }
-#define SIT_SELECTED SectorAddress { 0x0000, 0x0B13 }
-#define SIT_ACTIVATED SectorAddress { 0x0000, 0x0AC2 }
+#define SINGLE_STEP_4 SectorAddress { 0x0000, 0x30BB }
+#define SINGLE_STEP_4_SELECTED SectorAddress { 0x0000, 0x1627 }
+#define SINGLE_STEP_4_ACTIVATED SectorAddress { 0x0000, 0x1678 }
 
-#define HOME_SIT SectorAddress { 0x0000, 0x218A }
-#define HOME_SIT_SELECTED SectorAddress { 0x0000, 0x00A2 }
-#define HOME_SIT_ACTIVATED SectorAddress { 0x0000, 0x0051 }
+#define HOMESTAND_4_SSMENU SectorAddress { 0x0000, 0x310C }
+#define HOMESTAND_4_SSMENU_SELECTED SectorAddress { 0x0000, 0x180D }
+#define HOMESTAND_4_SSMENU_ACTIVATED SectorAddress { 0x0000, 0x185E }
 
-#define HOME_STAND SectorAddress { 0x0000, 0x21DB }
-#define HOME_STAND_SELECTED SectorAddress { 0x0000, 0x0288 }
-#define HOME_STAND_ACTIVATED SectorAddress { 0x0000, 0x0237 }
+#define SIT_4_SSMENU SectorAddress { 0x0000, 0x315D }
+#define SIT_4_SSMENU_SELECTED SectorAddress { 0x0000, 0x1900 }
+#define SIT_4_SSMENU_ACTIVATED SectorAddress { 0x0000, 0x1951 }
 
-#define TURN_OFF SectorAddress { 0x0000, 0x227D }
-#define TURN_OFF_SELECTED SectorAddress { 0x0000, 0x1491 }
-#define TURN_OFF_ACTIVATED SectorAddress { 0x0000, 0x1440 }
-/***** Sit menu *****/
+#define HOME_WAlKMENU SectorAddress { 0x0000, 0x31AE }
```

Comment:
And here it is defined again

Reply:
Yes that are the same pictures, so the first one can also have the value 31AE. But I made the gait selection in a sort of order of the whole gait selection and therefore sometimes the same pictures accidentally occurs.